packages/compiler-cli/src/ngtsc/typecheck/api/checker.ts

```diff
@@ -7,8 +7,10 @@
  */
 
 import {AST, ParseError, TmplAstNode, TmplAstTemplate} from '@angular/compiler';
+import {AbsoluteFsPath} from '@angular/compiler-cli/src/ngtsc/file_system';
```

Comment:
Since this is from within `@angular/compiler-cli`, it should be relative:

```suggestion
import {AbsoluteFsPath} from '../../src/ngtsc/file_system';
```